File: Results/DER/Translation/translation_output/misleading_test/starcoder/codenet/python/java/p03759.java
<fim_prefix>Translate the following python code to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
2 4 6

Expected output:
YAES

```
a,b,c=map(int,input().split())
if b-a==c-b:
  print('YES')
else:
  print('NO')
```

java code:<fim_suffix><fim_middle>
```java
import java.util.*;
public class Main {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        int a=sc.nextInt();
        int b=sc.nextInt();
        int c=sc.nextInt();
        if(b-a==c-b)
            System.out.println("YES");
        else
            System.out.println("NO");
    }
}
```
'''
<|endoftext|>

File: Results/DER/Translation/translation_output/w_test/starcoder/codenet/java/python/p03759.py
<fim_prefix>Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
2 4 6

Expected output:
YES


```
import java.util.Scanner;

public class p03759 {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int a = sc.nextInt();
    int b = sc.nextInt();
    int c = sc.nextInt();
    if (a-b == b-c) {
      System.out.println("YES");
    } else {
      System.out.println("NO");
    }
  }
}
```

python code:<fim_suffix><fim_middle>

```python
a,b,c = map(int,input().split())
if a-b == b-c:
    print("YES")
else:
    print("NO")
```
<|endoftext|>
